release/cmd/main.go

```diff
@@ -16,43 +16,45 @@ package main
 
 import (
 	"fmt"
-	"io"
 	"os"
+	"path/filepath"
 
 	"github.com/sirupsen/logrus"
 	cli "github.com/urfave/cli/v2"
-	"gopkg.in/natefinch/lumberjack.v2"
 
-	"github.com/projectcalico/calico/release/internal/config"
+	"github.com/projectcalico/calico/release/internal/command"
 	"github.com/projectcalico/calico/release/internal/utils"
 )
 
-var (
-	// debug controls whether or not to emit debug level logging.
-	debug bool
+type Config struct {
+	// RepoRootDir is the root directory for this repository
+	RepoRootDir string `envconfig:"REPO_ROOT"`
 
-	// releaseNotesDir is the directory where release notes are stored
-	releaseNotesDir = "release-notes"
-)
+	// OutputDir is the directory where all outputs are stored
+	OutputDir string
 
-func configureLogging(filename string) {
-	if debug {
-		logrus.SetLevel(logrus.DebugLevel)
-	} else {
-		logrus.SetLevel(logrus.InfoLevel)
-	}
+	// TmpDir is the directory for temporary files
+	TmpDir string
+}
 
-	// Set up logging to both stdout as well as a file.
-	writers := []io.Writer{os.Stdout, &lumberjack.Logger{
-		Filename:   filename,
-		MaxSize:    100,
-		MaxAge:     30,
-		MaxBackups: 10,
-	}}
-	logrus.SetOutput(io.MultiWriter(writers...))
+// loadConfig loads the configuration for the release tool.
+func loadConfig() (*Config, error) {
+	config := &Config{
+		RepoRootDir: os.Getenv("REPO_ROOT"),
```

Comment:
Should we make this one a flag as well? It's the last config field that gets configured externally and would allow us to get rid of this struct altogether I think (outputDir and tmpDir are derived from the config)

Reply:
No plus I do not think there is any value/ use case for making this externally configurable. I will update accordingly.

I am not sure why you want to get rid of this config struct as it is for internal configurations for the tool

Reply:
Config implies externally configurable to me is all.